client/shim.js

```diff
@@ -55,10 +55,14 @@
 
       m.runner = origRun.apply(mocha, arguments);
       if (m.runner.stats && m.runner.stats.end) {
-        window._eventbus.emit('ended', m.runner.stats);
+        setTimeout(()=>{
+          window._eventbus.emit('ended', m.runner.stats);
+        }, 2000)
```

Comment:
arbitrary timeouts are never a predictable or portable thing. two seconds is also a very long time. perhaps you're looking for a deferred promise?

Reply:
Sorry, I didn't realise that we made the PR from our forks master branch. The timeout was meant as a temp solution for our system where chrome gets shutdown before the reporter finishes.

It would make sense to close this PR and use the other open one as it contains the same fixes: https://github.com/shellscape/mocha-chrome/pull/45